source/macos/AdaptiveCards/AdaptiveCards/Utils/BundleUtils.swift

```diff
@@ -0,0 +1,25 @@
+import AppKit
+
+let bundleIdentifier = "com.test.test.AdaptiveCards"
+
+class BundleUtils {
+    static func getImage(_ imageName: String) -> NSImage? {
+        let index = imageName.lastIndex(of: ".") ?? imageName.endIndex
+        let fileType = String(imageName[imageName.index(after: index)...])
+        let name = String(imageName[...imageName.index(before: index)])
+        guard let bundle = Bundle(identifier: bundleIdentifier),
+              let path = bundle.path(forResource: name, ofType: fileType) else {
+            logError("Image Not Found")
+            return nil
+        }
+        return NSImage(byReferencing: URL(fileURLWithPath: path))
+    }
+    
+    static func loadNib(_ nibName: String, _ objectName: NSView) {
```

Comment:
```suggestion
    static func loadNibNamed(_ nibName: String, owner: NSView) {
```
We can use the same method definition.